src/gui/gui.cpp

```diff
@@ -1712,6 +1712,15 @@ void gui_ItemNotifier::Draw()
         base_item_p item = World_GetBaseItemByID(mItem);
         if(item)
         {
+            gl_text_line_p inventory_name = GLText_OutTextXY(mCurrPosX, 45.0f, "%s", item->name);
```

Comment:
upd: this time UI and core (font, text, console) modules is under heavy refactoring (it is better to reduce changes here before gui_refactoring branch merging).
inventory manager has it's own gl_text_line_p items